build.js

```diff
@@ -471,4 +477,41 @@ autobase.register({
   ]
 })
 
+autobase.register({
+  name: 'broadcast-encryption-key',
+  fields: [
+    {
+      name: 'id',
+      type: 'uint',
+      required: true
+    },
+    {
+      name: 'encryptionKey',
+      type: 'fixed32',
+      required: true
+    }
+  ]
+})
```

Comment:
Should we define this in `broadcast-encryption` maybe as a schema generator and import? Theoretically this struct could be used by anything using `broadcast-encryption`.